lib/commands/project/project.js

```diff
@@ -383,6 +380,41 @@ function testGenerate(directory, options, cb) {
   var testModule = options.testModule || TESTMODULES[0];
   var assertionFormat = options.assertionFormat || ASSERTIONTYPES[0];
 
+  findProjectFile(directory, null, function(err, projPath) {
+    var projectFile = fs.readFileSync(projPath, 'utf8');
```

Comment:
In general, that's not the best idea. Using require will not just read a JSON file, it will execute it.